CHANGELOG.md

```diff
@@ -5,9 +5,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [v1.16.0](https://github.com/flipt-io/flipt/releases/tag/v1.16.0) - 2022-11-30
+
+### Added
+
+- Automatic authentication background cleanup process [#1161](https://github.com/flipt-io/flipt/pull/1161).
+
+### Fixes
+
+- Dont log warnings when telemetry cannot report [#1156](https://github.com/flipt-io/flipt/pull/1156)
```

Comment:
should we include #1179 ?

Reply:
AHh yes
I thought we had that changelog entry in master already. My bad.